ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/ClusterSelection.tsx

```diff
@@ -163,6 +163,15 @@ function ClusterSelection({
             </PageSection>
             <Divider component="div" />
             <Form className="pf-v5-u-py-lg pf-v5-u-px-lg" ref={alertRef}>
+                <Alert
+                    title="At least one cluster must be in a Healthy state to proceed with the schedule."
+                    variant="info"
+                    isInline
+                >
+                    <p>
+                        💡 Tip: The most common reason a cluster is marked Unhealthy is that the Compliance Operator is either not installed or is below version 1.6.
```

Comment:
I, personally, like the emoji 👀. Do we use emojis in other places? If not, I wonder if we should remove it for now 🤔

Reply:
not sure, this copy came from maria. i can remove the emoji for now if we don't have other examples of that type of content? cc: @mansursyed